src/js/racingcar.js

```diff
@@ -0,0 +1,157 @@
+const initialState = {
+  raceCarList: [],
+  raceCnt: 0,
+};
+
+export default class racingCar {
+  constructor({
+    $raceCarListInp,
+    $raceCarListBtn,
+    $raceCntInp,
+    $raceCntBtn,
+    $raceGround,
+    $raceCntContainer,
+    $resultContainer,
+  }) {
+    this.$raceCarListInp = $raceCarListInp;
+    this.$raceCarListBtn = $raceCarListBtn;
+    this.$raceCntInp = $raceCntInp;
+    this.$raceCntBtn = $raceCntBtn;
+    this.$raceGround = $raceGround;
+    this.$raceCntContainer = $raceCntContainer;
+    this.$resultContainer = $resultContainer;
+    this.state = initialState;
+  }
+
+  bindEvents() {
+    this.$resultContainer.style.display = "none";
+    this.$raceCntContainer.style.display = "none";
+    this.$raceCarListBtn.addEventListener(
+      "click",
+      this.onClickRaceCarListBtn.bind(this)
+    );
+    this.$raceCntBtn.addEventListener(
+      "click",
+      this.onClickRaceCntBtn.bind(this)
+    );
+  }
+
+  onClickRaceCarListBtn() {
+    const splitCarList = this.$raceCarListInp.value
+      .split(",")
+      .map((ele) => ele.trim());
+    const isValue1_5 = splitCarList.every(
+      (raceCarName) => raceCarName.length >= 1 && raceCarName.length <= 5
+    );
+    if (isValue1_5) {
+      this.$raceCntContainer.style.display = "block";
+      return (this.state.raceCarList = splitCarList);
+    }
+    alert("자동차의 이름은 1자이상, 5자 이하만 가능합니다.");
+    this.state.raceCarList = [];
+  }
+
+  onClickRaceCntBtn() {
+    if (this.$raceCntInp.value < 1)
+      return alert(
+        "입력한 레이싱 횟수가 너무 적습니다. 레이싱 횟수는 1이상이어야 합니다."
+      );
+    this.state.raceCnt = Number(this.$raceCntInp.value);
+    this.showRacingCarGround();
+  }
+
+  getWinner($carPlayers, raceCarList) {
+    const $winnerTitle = document.querySelector("#winner-title");
+    const scoreList = Array.from($carPlayers).map(
+      ($player) => $player.children.length - 1 //이름칸인 div를 뺀 나머지 값이 결과값이다.
+    );
+    const maxScore = Math.max.apply(null, scoreList);
```

Comment:
```suggestion
    const maxScore = Math.max(...scoreList);
```

전개연산자를 활용해볼 수 있을 것 같습니다

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax